cmd/notary-server/config.json

```diff
@@ -13,6 +13,6 @@
 	},
 	"storage": {
 		"backend": "mysql",
-		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
+		"db_url": "gordontheturtle:dockerrulez@tcp(notarymysql:3306)/development"
```

Comment:
The format is fine, but the value are hard-coded.  What I mean was the connection string should reference the env variables like : "${DB_USER}:${DB_PASS}@"